src/anyon/qpu_interface.jl

```diff
@@ -279,7 +294,11 @@ function get_status(client::Client,circuitID::String)::Status
     end
 
     if body["status"]["type"]==failed_status
-        return Status(type=body["status"]["type"],message=body["message"])
+        try
+            return Status(type=failed_status,message=body["message"])
+        catch
+            return Status(type=failed_status,message=string(body))
```

Comment:
I think the message should be "no failure information available". Since this will correspond to a response of
`{"type": "failed"}`, which shouldn't happen, but it is still valid according to our OpenAPI spec.

Also, we need a test for this.

Reply:
For debug-ability, I like including a dump of the raw response in these "should not happen" cases
:thinking: 
What if we include both? Something like `no failure information available. raw response: '${string(body)}'`?